Objects (Table, column. index) cannot be created via UI

#### System information: 
- Operating system (distribution) and version win10
- DBeaver version 22.0.1

#### Connection specification:
- Database name and version Apache Druid

#### Describe the problem you're observing:
It is impossible to create table, column, index, fk via UI.

Error on table creation:
<details>
```
eclipse.buildId=unknown
java.version=11.0.12
java.vendor=Eclipse Foundation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Command-line arguments:  -os win32 -ws win32 -arch x86_64

org.jkiss.dbeaver.model
Error
Thu Mar 24 13:28:03 MSK 2022
Error -1 (00000) : Error while executing SQL "CREATE TABLE "druid"."druid".NEWTABLE (
	COLUMN1 VARCHAR(100)
)": Remote driver error: RuntimeException: org.apache.calcite.sql.parser.SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
     -> SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
     -> ParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    

org.apache.calcite.avatica.AvaticaSqlException: Error -1 (00000) : Error while executing SQL "CREATE TABLE "druid"."druid".NEWTABLE (
	COLUMN1 VARCHAR(100)
)": Remote driver error: RuntimeException: org.apache.calcite.sql.parser.SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
     -> SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
     -> ParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    
	at org.apache.calcite.avatica.Helper.createException(Helper.java:54)
	at org.apache.calcite.avatica.Helper.createException(Helper.java:41)
	at org.apache.calcite.avatica.AvaticaStatement.executeInternal(AvaticaStatement.java:163)
	at org.apache.calcite.avatica.AvaticaStatement.execute(AvaticaStatement.java:217)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:329)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.lambda$0(JDBCStatementImpl.java:131)
	at org.jkiss.dbeaver.utils.SecurityManagerUtils.wrapDriverActions(SecurityManagerUtils.java:94)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.executePersistAction(DBExecUtils.java:314)
	at org.jkiss.dbeaver.model.impl.edit.AbstractObjectManager.executePersistAction(AbstractObjectManager.java:40)
	at org.jkiss.dbeaver.model.impl.edit.AbstractCommandContext.executeCommands(AbstractCommandContext.java:205)
	at org.jkiss.dbeaver.model.impl.edit.AbstractCommandContext.saveChanges(AbstractCommandContext.java:121)
	at org.jkiss.dbeaver.ui.editors.entity.EntityEditor.lambda$1(EntityEditor.java:360)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:171)
	at org.jkiss.dbeaver.ui.editors.entity.EntityEditor.saveCommandContext(EntityEditor.java:358)
	at org.jkiss.dbeaver.ui.editors.entity.EntityEditor$SaveJob.run(EntityEditor.java:1198)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
java.lang.RuntimeException: org.apache.calcite.sql.parser.SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    
	at org.apache.druid.sql.avatica.DruidStatement.closeAndPropagateThrowable(DruidStatement.java:377)
	at org.apache.druid.sql.avatica.DruidStatement.prepare(DruidStatement.java:181)
	at org.apache.druid.sql.avatica.DruidMeta.prepareAndExecute(DruidMeta.java:191)
	at org.apache.calcite.avatica.remote.LocalService.apply(LocalService.java:206)
	at org.apache.calcite.avatica.remote.Service$PrepareAndExecuteRequest.accept(Service.java:927)
	at org.apache.calcite.avatica.remote.Service$PrepareAndExecuteRequest.accept(Service.java:879)
	at org.apache.calcite.avatica.remote.AbstractHandler.apply(AbstractHandler.java:94)
	at org.apache.calcite.avatica.remote.JsonHandler.apply(JsonHandler.java:52)
	at org.apache.calcite.avatica.server.AvaticaJsonHandler.handle(AvaticaJsonHandler.java:133)
	at org.apache.druid.sql.avatica.DruidAvaticaHandler.handle(DruidAvaticaHandler.java:59)
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:59)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:179)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:516)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:773)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:905)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.apache.calcite.sql.parser.SqlParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.convertException(SqlParserImpl.java:348)
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.normalizeException(SqlParserImpl.java:147)
	at org.apache.calcite.sql.parser.SqlParser.handleException(SqlParser.java:147)
	at org.apache.calcite.sql.parser.SqlParser.parseQuery(SqlParser.java:162)
	at org.apache.calcite.sql.parser.SqlParser.parseStmt(SqlParser.java:187)
	at org.apache.calcite.prepare.PlannerImpl.parse(PlannerImpl.java:201)
	at org.apache.calcite.tools.Planner.parse(Planner.java:50)
	at org.apache.druid.sql.calcite.planner.DruidPlanner.prepare(DruidPlanner.java:97)
	at org.apache.druid.sql.SqlLifecycle.prepare(SqlLifecycle.java:155)
	at org.apache.druid.sql.avatica.DruidStatement.prepare(DruidStatement.java:162)
	... 27 more
Caused by: org.apache.calcite.sql.parser.impl.ParseException: Encountered "CREATE" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "ELEMENT" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.generateParseException(SqlParserImpl.java:31029)
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.jj_consume_token(SqlParserImpl.java:30846)
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.SqlStmt(SqlParserImpl.java:3583)
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.SqlStmtEof(SqlParserImpl.java:3595)
	at org.apache.calcite.sql.parser.impl.SqlParserImpl.parseSqlStmtEof(SqlParserImpl.java:194)
	at org.apache.calcite.sql.parser.SqlParser.parseQuery(SqlParser.java:160)
	... 33 more

```
</details>

--- a/plugins/org.jkiss.dbeaver.ext.generic/plugin.xml
+++ b/plugins/org.jkiss.dbeaver.ext.generic/plugin.xml
@@ -1201,6 +1201,8 @@
                     <parameter name="supports-indexes" value="false"/>
                     <parameter name="supports-references" value="false"/>
                     <parameter name="supports-table-constraints" value="false"/>
+                    <parameter name="read-only-data" value="true"/>
+                    <parameter name="read-only-meta-data" value="true"/>
                 </driver>
 
                 <!-- Apache Kylin -->
@@ -1223,6 +1225,8 @@
                     <parameter name="supports-indexes" value="false"/>
                     <parameter name="supports-references" value="false"/>
                     <parameter name="supports-table-constraints" value="false"/>
+                    <parameter name="read-only-data" value="true"/>
+                    <parameter name="read-only-meta-data" value="true"/>
                 </driver>
 
 

